Source/SwiftLintFramework/Rules/UnusedOptionalBindingRule.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import SourceKittenFramework
 
 public struct UnusedOptionalBindingRule: ASTRule, ConfigurationProviderRule {
-    public var configuration = SeverityConfiguration(.warning)
+    public var configuration = UnusedOptionalBindingConfiguration(ignoreOptionalTry: true)
```

Comment:
I don't have strong opinions here, so I'm ok with reverting the default to `false`